_includes/pbc/all/install-features/latest/install-self-service-portal.md

```diff
@@ -1,17 +1,9 @@
-
-
-
 {% info_block warningBox %}
 
 Self-Service Portal is currently running under an Early Access Release. Early Access Releases are subject to specific legal terms, they are unsupported and do not provide production-ready SLAs. They can also be deprecated without a General Availability Release. Nevertheless, we welcome feedback from early adopters on these cutting-edge, exploratory features.
```

Comment:
please confirm that this information should stay, as it was removed from bottom pages